packages/rx_bloc_cli/README.md

```diff
@@ -49,6 +49,7 @@ Create a new project at the given directory with lots of already set-up features
 | `--[no-]interactive`                   |                   `false`                   |              Enables interactive project generation flow flags are read from command line prompts               |
 | `--[no-]enable-pin-code`               |                   `false`                   |            Enables pin code feature with biometrics, that can help with building pin code workflows.            |
 | `--cicd`                               |                 `fastlane`                  | Provides a template for setting up ci/cd for your project. Available options: none, fastlane, github, codemagic |
+| `--[no-]enable-feature-qr-scanner`     |                 `false`                     |                              Enables QR scanner feature for the project                                  |
```

Comment:
I find the idea of adding feature as part of the command appropriate but from other side is inconsistent with the rest of the showcase feature. With that being said I think that it needs to be changed to --[no-]enable-qr-scanner